examples/sensors-viewer/src/components/SensorItem.jsx

```diff
@@ -0,0 +1,64 @@
+import {Accordion, Button, Card, ListGroup} from "react-bootstrap";
+import {Images} from "../Images";
+import React from "react";
+
+const _ = require('lodash');
+const moment = require('moment');
+
+function listItem(item, index) {
+    return <ListGroup.Item key={index}
+                           className="border-0 px-0 py-2 font-weight-normal text-capitalize bg-transparent">
+        <span className="pr-2 text-dark font-weight-bold">{item.label}:</span>
+        {item.value}
+    </ListGroup.Item>
+}
+
+export default function SensorItem({availableSensors, sensorValues, sensorSamplingRate}) {
+    return (
+        Object.keys(sensorValues).map((item, index) => {
```

Comment:
this `.map` function can be moved outside the first `return`

```
const myModel = /* some transformation */;

return myModel.map( renderSingleItemFunction ); // or myModel.forEach depending on your model type
```

Reply:
Can you give me more description about this?